.github/workflows/deploy-worker.yml

```diff
@@ -115,3 +125,36 @@ jobs:
           wranglerVersion: ${{ env.wranglerVersion }}
           workingDirectory: ${{ github.event.inputs.directory }}
           command: deploy --no-bundle --name=${{ github.event.inputs.appId }} --dispatch-namespace ${{ github.event.inputs.dispatchNamespace }} ${{ steps.get-script.outputs.WORKER_SCRIPT }}
+
+  attest:
+    needs: deploy
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      attestations: write
+
+    steps:
+      - name: Download worker script
+        uses: actions/download-artifact@v4
+        with:
+          name: worker_script
+          path: ./
+
+      - uses: actions/attest-build-provenance/predicate@d58ddf9f241cd8163408934540d01c3335864d64 # predicate@1.1.2
+        id: generate-build-provenance-predicate
+
+      - name: Update Predicate JSON
+        id: update-predicate
+        run: |
+          uri="git+https://github.com/${{ github.event.inputs.repo }}@refs/heads/${{ github.event.inputs.branch }}"
+          resolved_dependencies=$(jq -n --arg uri "$uri" --arg commit "${{ github.event.inputs.commit }}" '[{"uri": $uri, "digest": {"gitCommit": $commit}}]')
+          predicate=$(echo '${{ steps.generate-build-provenance-predicate.outputs.predicate }}' | jq -c '.buildDefinition.externalParameters.resolvedDependencies = $resolved_dependencies' --argjson resolved_dependencies "$resolved_dependencies")
```

Comment:
Overriding the existing `.buildDefinition.resolvedDependencies` resulted in
```
Error: Failed to persist attestation: Invalid Argument - values do not match: git+https://github.com/wilsonianb/php-worker-hello-world@refs/heads/master != git+https://github.com/codius/codius-workers@refs/heads/preview - https://docs.github.com/rest/repos/repos#create-an-attestation
```
which appears to originate in
https://github.com/actions/toolkit/blob/6c4e082c181a51609197e536ef5255a0c9baeef7/packages/attest/src/store.ts#L28-L42

Specifying the worker's repo in an added `resolvedDependencies` field under `externalParameters` appears to work.
However, it isn't displayed in either the [GitHub attestation](https://github.com/codius/codius-workers/attestations/1774723) or [Rekor entry](https://search.sigstore.dev/?logIndex=123971008)

It can be found by parsing the attestation file:
```
jq -r '.dsseEnvelope.payload' codius-codius-workers-attestation-1774723.sigstore.json | base64 -d | jq .
```

https://slsa.dev/spec/v1.0/provenance#builddefinition